packages/core/src/dispatcher.ts

```diff
@@ -311,6 +323,7 @@ export class Dispatcher {
    */
   async close(): Promise<void> {
     this.messageBus.unsubscribe()
+    await this.tasks.onIdle()
```

Comment:
oooh, does this fix https://github.com/ceramicnetwork/js-ceramic/issues/1963?

Reply:
yes.